src/components/Graph.js

```diff
@@ -30,6 +30,14 @@ export class Graph extends React.Component {
       nodeCircle: null,
       nodeIcon: null,
       nodeGroup: null,
+      nodeData: {
+        kind: '',
```

Comment:
I have a comment connected with ``kind`` value. So far support only two types of data in Node Detail Card:
 - object detail info
 - statistics

Maybe we should use boolean instead string in this case?